src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/HearingPutConsumerTest.java

```diff
@@ -0,0 +1,213 @@
+            .uponReceiving("Request to UPDATE hearing for forbidden hearing request")
+            .path(ContractTestDataProvider.PATH_HEARING).method(HttpMethod.PUT.toString())
+            .query(ContractTestDataProvider.FIELD_ID + "=" + ContractTestDataProvider.FORBIDDEN_CASE_ID)
+            .body(ContractTestDataProvider.toJsonString(ContractTestDataProvider.generateHearingRequest()))
+            .headers(ContractTestDataProvider.headers)
+            .willRespondWith().status(HttpStatus.FORBIDDEN.value())
+            .body(new PactDslJsonBody()
+                      .stringType(ContractTestDataProvider.FIELD_MESSAGE,ContractTestDataProvider.MSG_403_HEARING)
+                      .stringValue(ContractTestDataProvider.FIELD_STATUS, HttpStatus.FORBIDDEN.value()
+                          + " " + HttpStatus.FORBIDDEN.getReasonPhrase())
+                      .eachLike(ContractTestDataProvider.FIELD_ERRORS, 1)
+                      .closeArray())
+            .toPact();
+    }
+
+    @Pact(consumer = ContractTestDataProvider.CONSUMER_NAME)
+    public RequestResponsePact notFoundRequestErrorFromPutHearing(PactDslWithProvider builder) {
+        return builder.given(ContractTestDataProvider.CONSUMER_NAME
+                                 + " throws not found request error while trying to update hearing")
+            .uponReceiving("Request to UPDATE hearing for not found hearing request")
+            .path(ContractTestDataProvider.PATH_HEARING).method(HttpMethod.PUT.toString())
+            .query(ContractTestDataProvider.FIELD_ID + "=" + ContractTestDataProvider.NOT_FOUND_CASE_ID)
+            .body(ContractTestDataProvider.toJsonString(ContractTestDataProvider.generateHearingRequest()))
+            .headers(ContractTestDataProvider.headers)
+            .willRespondWith().status(HttpStatus.NOT_FOUND.value())
+            .body(new PactDslJsonBody()
+                      .stringType(ContractTestDataProvider.FIELD_MESSAGE,ContractTestDataProvider.MSG_404_HEARING)
+                      .stringValue(ContractTestDataProvider.FIELD_STATUS, HttpStatus.NOT_FOUND.value()
+                          + " " + HttpStatus.NOT_FOUND.getReasonPhrase())
+                      .eachLike(ContractTestDataProvider.FIELD_ERRORS, 1)
+                      .closeArray())
+            .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "updateHearingRequestForValidRequest")
+    public void shouldSuccessfullyPutHearingRequest() throws JsonProcessingException {
+        HearingResponse hearingResponse = hmcHearingApi.updateHearingRequest(
+            ContractTestDataProvider.IDAM_OAUTH2_TOKEN,
+            ContractTestDataProvider.SERVICE_AUTHORIZATION_TOKEN,
+            ContractTestDataProvider.VALID_CASE_ID,
+            ContractTestDataProvider.generateHearingRequest()
+        );
+
+        assertTrue(hearingResponse.getHearingRequestId() > ContractTestDataProvider.ZERO_LENGTH);
```

Comment:
Can you please provide concrete assertions (the actual values you expect, not that it should be > 0 or !=0 )